src/components/misc/personViews/PersonViewTable.jsx

```diff
@@ -128,19 +128,36 @@ export default class PersonViewTable extends React.Component {
 
                     // Sort, if a column is selected for sorting
                     if (this.state.sortIndex !== null) {
-                        visibleRows = visibleRows.concat().sort((row0, row1) => {
-                            const val0 = row0.data.content[this.state.sortIndex] || '';
-                            const val1 = row1.data.content[this.state.sortIndex] || '';
+                            visibleRows = visibleRows.concat().sort((row0, row1) => {
```

Comment:
The indentation on this line (and hence all subsequent lines in this PR) is wrong. This line should be indented four spaces relative to the line before it, but is currently indented 8 spaces.

This means that all subsequent lines down to and including line 160 are indented four spaces too much.